tests/end2end/test_invocations.py

```diff
@@ -847,6 +855,12 @@ def test_sandboxing(
     quteproc_new.open_url('chrome://sandbox')
     text = quteproc_new.get_content()
     print(text)
+
+    not_found_msg = ("The webpage at chrome://sandbox/ might be temporarily down or "
+                     "it may have moved permanently to a new web address.")
```

Comment:
Windows and macOS - it looks like [QtWebEngine only supports](https://codereview.qt-project.org/c/qt/qtwebengine/+/205250) `chrome://sandbox` on Linux, probably because on Windows the page looks different (I'm guessing an entirely different code path):

![image](https://user-images.githubusercontent.com/625793/157011017-41913070-2dfb-4107-b2a3-e518ef1d5727.png)

(can't test macOS right now, for whatever reason, my Mac doesn't seem to be reachable via VNC anymore...).

So I suppose if this changes in the future, the test will start failing, and we will notice.